### ISSUE TITLE
Application not working.

### ISSUE THREAD

**Issue by illegalblue** [WolfSource/Wolf.Playout.Builds#2]
![screenshot 9](https://user-images.githubusercontent.com/29424127/36252036-61d7949a-1269-11e8-8134-e87c17819830.png)

OS : win 10 home x64
RAM : 4 GB

Im testing on my home computer. But the application is not working.
I

**Comment by illegalblue** [comment]
[Log.zip](https://github.com/WolfSource/Wolf.Playout.Builds/files/1731959/Log.zip)


PFA.  Uploaded the logs!

**Comment by PooyaEimandar** [comment]
it seems that, application could not find BlackMagic DeckLink drivers, did you setup BlackMagic Desktop Video 10.5.2 on your target machine?

**Comment by illegalblue** [comment]
No.  I have the drivers and device set up on another device with windows XP. 

I just wanted to test the UI,  so checked on my personal computer which has no drivers. 

Let me check the software on the fully setup machine. And will update the logs. 
Thank you for the response.

**Comment by illegalblue** [comment]
Procedure entry point CreateFile2 could not be located in the dynamic link library KERNEL32. 

This error is with Win7. I think createfile2 entry point is from Win8. Isn't it?
Whats the minimum requirement for this software to run!?
Please advise.

**Comment by PooyaEimandar** [comment]
Please find prerequisites of Playout here:
https://github.com/WolfSource/Wolf.Playout.Builds#prerequisites

Windows 8.1 or 10 (recommended) are supported. Sorry windows 7 is not supported yet, I going to support it soon.
